src/stratis_cli/_errors.py

```diff
@@ -161,23 +161,32 @@ class StratisCliIncoherenceError(StratisCliRuntimeError):
 
 
 class StratisCliInUseError(StratisCliUserError):
+    """
+    Base class for if a request made of stratisd must result in a device being
+    included in both data and cache tiers or in the same tier of two different
+    pools.
+    """
+
+
+class StratisCliInUseOtherTierError(StratisCliInUseError):
     """
     Raised if a request made of stratisd must result in a device being
     included in both data and cache tiers.
     """
 
-    def __init__(self, blockdevs, added_as):
+    def __init__(self, pools_to_blockdevs, added_as):
         """ Initializer.
 
-            :param blockdevs: the blockdevs that would be added in both tiers
-            :type blockdevs: frozenset of str
+            :param pools_to_blockdevs: pools mapped to the blockdevs they own
+            :type pools_to_blockdevs: dict of dbus.types.ObjectPath * frozenset of str
```

Comment:
Whoops! Looks like I got a little tripped up because of the return type being a D-Bus type. I'm adding a fix to convert it from a dbus library string type to a `str` as well.

Reply:
Resolved.